MeetingScheduler/Logger/LogHelper.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    public static class LogHelper
+    {
+        private static LogBase logger = null;
+        public static void Log(LogTarget target, string message)
+        {
+            switch (target)
+            {
+                case LogTarget.File:
```

Comment:
Da li je neophodno na svaki poziv da se pravi novi objekat svakog loggera? Zamisli koliko k linija koda i upisa ima prosecan log fajl u satu npr.

Reply:
Dodatno razmisliti o factory desing patternu ovde primeniti ako si se opredelio za rukovanje na ovaj nacin (sto je ok)

Reply:
Prepravio sam kod, tako da se koristi factory design pattern. Ovak oe se formirati novi logovi samo kada se to zatrazi, a ne pri svakom logovanju neke linije.